### DISCUSSION TITLE
Include statically generated info (api docs/javadocs) in docusaurus

### DISCUSSION THREAD

**Discussion by jyothsnakanand** [facebook/docusaurus#8479] [Q&A]
I'd like to add statically generated content (say api docs/javadocs) into docusaurus. I've tried using iframes to load the content but it cannot find it. 

The generated javadocs are available in `/static/<submodule>/*` - contains html/css/js generated by the javadocs utility

Within `/docs/`, I'd like to create one page each with an iframe that loads this content but I think I'm missing something here. the iframe loads the landing page for my documentation everytime.

**Comment by slorber** [comment]
Hi

Please deploy a POC somewhere where I can inspect the URL and why it fails, otherwise it will be really complicated and time consuming for me to help you.

For example, I can't check your deployment so I can't be sure you didn't make any typo in the iframe URL. Possible error: the iframe URL should not include `/static/<submodule>/page.html` but just be `/<submodule>/page.html`.

**Reply by jyothsnakanand** [reply]
Sure! I'll host a POC shortly. In the meantime, I've attached screenshots and contents of `/static/<submodule>/` and the markdown file `docs/API Documentation/hdfs-monitor-service-factory.md`

Screenshot of the page deployed locally: 
<img width="1724" alt="Screenshot 2022-12-23 at 11 40 52 AM" src="https://user-images.githubusercontent.com/8680508/209399244-02a771be-95bf-471b-81fc-9af6fd77f23b.png">


List of files in `/static/<submodule>/` (basically generated html from javadocs) - 
/static/hdfs-monitor-service-factory$ ls
allclasses-frame.html
allclasses-noframe.html
com
constant-values.html
deprecated-list.html
help-doc.html
index-all.html
index.html
overview-tree.html
package-list
script.js
stylesheet.css


The markdown file `hdfs-monitor-service-factory.md` just contains an iframe after the sidebar position: 

//---
sidebar_position: 1
//---

<iframe src='/hdfs-monitor-service-factory/index.html' width='140%' height='6000'> </iframe>

**Reply by jyothsnakanand** [reply]
the src field in the iframe isnt able to resolve this path to `/hdfs-monitor-service-factory/index.html`. 
I inspected the build folder and it contains `/hdfs-monitor-service-factory/*` - What am i missing?

**Reply by slorber** [reply]
I can't help without the ability to inspect both the code and the preview. Please provide both and we'll help
